bin/pycbc_inspiral

```diff
@@ -283,7 +290,21 @@ with ctx:
         q_trans = {}
 
     # FIXME: Maybe we should use the PSD corresponding to each trigger
-    event_mgr = events.EventManager(
+    if opt.checkpoint_interval:
+        checkpoint_file = opt.output + '.checkpoint'
+
+    checkpoint_exists = False
+    tnum_start = 0
+    if (opt.checkpoint_interval and os.path.isfile(checkpoint_file)):
+        try:
+            tnum_start, event_mgr = events.EventManager.restore_state(checkpoint_file)
+            checkpoint_exists = True
+        except Exception as e:
+            logging.info('Failed to load checkpoint file, starting anew')
+            logging.info(e)
```

Comment:
This behaviour could be dangerous? Maybe add an option to just fail if the checkpoint is there but not valid.

Reply:
Hmm.. I need to think about that. The default should be as is in any case as you don't want a filesystem problem to cause an infinite failure loop.

Reply:
Ok, I've added a --require-valid-checkpoint option, but it is False by default.